src/amo/components/CollectionManager/index.js

```diff
@@ -179,6 +200,24 @@ export class CollectionManagerBase extends React.Component<Props, State> {
             value={this.state.slug}
           />
         </div>
+        <AutoSearchInput
+          inputName="collection-addon-query"
+          inputPlaceholder={
+            i18n.gettext('Enter the name of an add-on to include')
```

Comment:
I'm not sure if this text was in the mocks but I find it a bit awkward; it's a sentence fragment and while it's likely got enough context around it to make sense of it might be tough for translators ("include in what?").

I think something like "Find an add-on to include in this collection" might work better.